pkg/command/cmd_handler.go

```diff
@@ -13,8 +13,8 @@ type CommentCmdHandler struct {
 }
 
 // Register adds the given CommentCmd implementation to the list of commands to be handled when an IssueCommentEvent occurs
-func (s *CommentCmdHandler) Register(command CommentCmd) {
-	s.commands = append(s.commands, command)
+func (s *CommentCmdHandler) Register(commands ...CommentCmd) {
+	s.commands = append(s.commands, commands...)
```

Comment:
The intention was to call the `Register` method for every command so the code is not so chaotic as is now https://github.com/arquillian/ike-prow-plugins/pull/147/files#diff-a87c6e95680b0dbcaec67f13a53e49adR104 it's hard to find where the previous command definition ends and the new one starts. Otherwise, there is no point in appending it :-). Just my point of view ;-)